### ISSUE TITLE
No Reply from Server 

### ISSUE THREAD

**Issue by noorbakerally** [cavendish-ldp/cavendish#18]
When creating a resource with an empty Slug, the server doesn't replies and the connection stays open and nothing happens, is that behaviour normal ?

**Comment by barmintor** [comment]
No, I would not expect that. There's code to generate a slug in the LdpHandler, and unless I misremember, that's exercised in the LDP test suite. Can you describe the HTTP request that's causing this behavior?
